parquet-scrooge/src/test/thrift/test.thrift

```diff
@@ -169,6 +169,95 @@ struct TestFieldOfEnum{
 }
 
 struct StringAndBinary {
-  1: required string s;
-  2: required binary b;
+  1: required string s
+  2: required binary b
+}
+
+#fixture fox nested structures
+struct NestedList {
```

Comment:
What do you mean? For the thrift file, we generate both scrooge and tbase classes. And they use the same same thrift definition, so we can compare the schema derived from a thrift class and a scrooge class